### DISCUSSION TITLE
Multiple Hostnames (Custom Domains)

### DISCUSSION THREAD

**Discussion by iammattmartin** [plausible/analytics#676] [Self-Hosted Support]
We've installed at metrics.domain.com and want to point unique sub-domains to that such as metrics.brand1.com etc.

This works and the tracking is picked up and all is great. 

Although we're still seeing the original name being generated in the JS:

```
(window,"https://metrics.domain.com");
```

Which appears to cause hits to "/api/event" go to that name instead.

Is there anyway to get Plausible to use the referenced script host instead (i.e `<script async defer data-domain="brand1.com" src="https://metrics.brand1.com/js/plausible.js"></script>` ?

**Comment by ZackBoe** [comment]
It looks like this works now? I've setup an instance (v1.5) and hosted it on two separate domains - both of which appear to be working simultaneously. In the generated script.js I'm seeing the following, so it should use the domain from the script's `src` or a full events path included in the `data-api` attribute (mentioned in the docs for proxying through Cloudflare/Netlify/Vercel/etc). I haven't tested with any script extensions, however.
```js
s = o.getAttribute('data-api') || new URL(o.src).origin + '/api/event';
```
